README.md

```diff
@@ -12,13 +12,8 @@ This package was renamed to `mongodb/laravel-mongodb` because of a transfer of o
 It is compatible with Laravel 10.x. For older versions of Laravel, please refer to the
 [old versions](https://github.com/mongodb/laravel-mongodb/tree/3.9#laravel-version-compatibility).
 
-- [Installation](https://www.mongodb.com/docs/drivers/php/laravel-mongodb/current/install/)
-- [Eloquent Models](https://www.mongodb.com/docs/drivers/php/laravel-mongodb/current/eloquent-models/)
-- [Query Builder](https://www.mongodb.com/docs/drivers/php/laravel-mongodb/current/query-builder/)
-- [Transactions](https://www.mongodb.com/docs/drivers/php/laravel-mongodb/current/transactions/)
-- [User Authentication](https://www.mongodb.com/docs/drivers/php/laravel-mongodb/current/user-authentication/)
-- [Queues](https://www.mongodb.com/docs/drivers/php/laravel-mongodb/current/queues/)
-- [Upgrading](https://www.mongodb.com/docs/drivers/php/laravel-mongodb/current/upgrade/)
+## Documentation
+Reference documentation for the Laravel MongoDB package is available [here](https://www.mongodb.com/docs/drivers/php/laravel-mongodb/).
```

Comment:
Link on "here" text is not recommended. 

Maybe something like this (not sure):
```suggestion
Read the reference documentation for the Laravel MongoDB package [on MongoDB website](https://www.mongodb.com/docs/drivers/php/laravel-mongodb/).
```

Reply:
Thanks, makes sense.